jac-uk/apply
# Feature/1066 Capture commissioner conflicts

## What's included?
Closes #1066 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Example vacancy: https://jac-apply-develop--pr1087-feature-1066-capture-s74e5f8l.web.app/vacancy/ONqUjAzhDS7rjh6dFJLB/

1. Apply for the vacancy.
2. Complete the "Commissioner Conflicts" section.
3. Check if the answers are saved on the review page.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/jac-uk/apply/assets/79906532/347f6dce-fc16-41c3-9058-4fc33b09d73d

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
